local_server/local-server-feature-layer/README.md

```diff
@@ -1,36 +1,35 @@
+A Local Server and Local Feature Service will automatically be started. Once started then a `FeatureLayer` will be created and added to the map.
 
 ## How it works
 
-To create a `FeatureLayer` from a `LocalFeatureService`:
-
-1. Create and run a local server.
-    * `LocalServer.INSTANCE` creates a local server
-    * `Server.startAsync()` starts the server asynchronously
-2. Wait for server to be in th. `LocalServerStatus.STARTED` state.
-    * `Server.addStatusChangedListener()` fires whenever the status of the local server has changed.
-3. Create and run a local feature service.
-    * `new LocalFeatureService(Url)`, creates a local feature service with the given url path to mpk file
-    * `LocalFeatureService.startAsync()`, starts the service asynchronously
-    * service will be added to the local server automatically
-4. Wait for feature service to be in th. `LocalServerStatus.STARTED` state.
-    * `LocalFeatureService.addStatusChangedListener()` fires whenever the status of the local service has changed.
-5. Create a feature layer from local feature service.
-    * create a `ServiceFeatureTable(Url)` from local feature service url, `LocalFeatureService.getUrl()`
-    * load the table asynchronously, `ServiceFeatureTable.loadAsync()`
-    * create feature layer from service feature table, `new FeatureLayer(ServiceFeatureTable)`
-    * load the layer asynchronously, `FeatureLayer.loadAsync()`
-6. Add feature layer to map, `Map.getOperationalLayers().add(FeatureLayer)`.
+1. Create and run a local server with `LocalServer.instance`.
+2. Start the server asynchronously with `Server.startAsync()`.
+3. Wait for server to be in the  `LocalServerStatus.STARTED` state.
+   * Callbacks attached to `Server.addStatusChangedListener()` will invoke whenever the status of the local server has changed.
+4. Create and run a local feature service.
+   1. Instantiate `LocalFeatureService(Url)` to create  a local feature service with the given url path to mpk file.
+   2. Start the service asynchronously with `LocalFeatureService.startAsync()`.
+      * The service will be added to the local server automatically.
+5. Wait for state of the feature service to be `LocalServerStatus.STARTED`.
+   * Callbacks attached to `LocalFeatureService.addStatusChangedListener()` will invoke whenever the status of the local service has changed.
+6. Create a feature layer from local feature service.
+   1. Create a `ServiceFeatureTable(Url)` from local feature service url provided by `LocalFeatureService.getUrl()`.
+   2. Load the table asynchronously using `ServiceFeatureTable.loadAsync()`.
+   3. Create feature layer from service feature table using `new FeatureLayer(ServiceFeatureTable)`.
+   4. Load the layer asynchronously using `FeatureLayer.loadAsync()`.
+7. Add feature layer to map using `Map.getOperationalLayers().add(FeatureLayer)`.
```

Comment:
we should refer to the official name of our map class API here since it's being used for the first time 
```suggestion
7. Add feature layer to map using `ArcGISMap.getOperationalLayers().add(FeatureLayer)`.
```

Reply:
Tihs trips me up so often, thanks for paying so much attention to these!